sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -581,6 +581,10 @@ ALTER TABLE `patient_data` ADD `uuid` binary(16) DEFAULT NULL;
 CREATE UNIQUE INDEX `uuid` ON `patient_data` (`uuid`);
 #EndIf
 
+#IfMissingColumn uuid_registry table_vertical
+ALTER TABLE `uuid_registry` ADD `table_vertical` varchar(255) NOT NULL DEFAULT '';
+#EndIf
+
```

Comment:
How in the world have we not seen this in community!

Reply:
@sjpadgett The only places that would have triggered this is facility_user_ids and practitioner_role before we added it to history_data.  Perhaps those tables were empty before?

Reply:
@sjpadgett Also we throw exceptions now so that we fail fast and hard in the UUID code.  If there was data and it was failing before, we may have just been having the errors suppressed.

Reply:
That'd be my guess too. We prob should look into that because some of the forum issues with upgrade may be related.

Reply:
Very nice that the demo with data picked this up (note this demo is basically doing an upgrade from 5.0.0 to the master via standard upgrade scripts). This bug will only show in users that are upgrading from pre 6.0.0 to 6.1.0-dev (ie. current dev version), so should not be an issue for current production 6.0.0.
Rec (so it is more naturally placed) to move it up a little bit after the `#IfMissingColumn uuid_registry document_drive` block